.github/scripts/auto-label.cjs

```diff
@@ -23,6 +23,10 @@ const prefixLabelMap = {
   '.eslintrc.js': 'style',
   '.github/workflows/': 'ci',
   '.github/scripts/': 'ci',
+  'plugin/': 'plugin',
+  'docs/': 'docs',
```

Comment:
Should the various ko-KR/, zh-CN/, zh-TW/ subdirectories get something for translations?

Reply:
Done, although having the "only the first match applies" and then "this can apply multiple labels" made things slightly more complicated, but I think it's ok.